assets/less/zenodo-rdm/globals/site.overrides

```diff
@@ -106,3 +106,6 @@ a.inverted {
   height: 0.85em;
   width: 1.2em;
 }
+.uppercase {
+  text-transform: uppercase !important;
```

Comment:
invenio-theme candidate?

Reply:
I wouldn't put it if that's only for this case as I think is a hack to uppercase the values...but indeed seems like a generic class to add in invenio-theme!